product_template_tags/views/product_template_tag.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="product_template_tag_form_view">
+        <field name="name">product.template.tag.form (in product_template_tags)</field>
+        <field name="model">product.template.tag</field>
+        <field name="arch" type="xml">
+            <form>
+                <header/>
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <button name="%(product.product_template_action)d"
+                                type="action"
+                                class="oe_stat_button"
+                                icon="fa-archive"
+                                context="{'search_default_tag_ids': active_id}">
+                            <field name="products_count" widget="statinfo"/>
+                        </button>
+                    </div>
+
+                    <group>
+                        <field name="name"/>
+                        <field name="company_id"
+                               option="{no_create_edit: True}"
+                               groups="base.group_multi_company"/>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="product_template_tag_search_view">
```

Comment:
just aesthetic change:
id before model

id pattern:
For a view: <model_name>_view_<view_type>, where view_type is kanban, form, tree, search, ...